crates/rust-analyzer/src/config.rs

```diff
@@ -321,14 +321,14 @@ config_data! {
         notifications_cargoTomlNotFound: bool      = "true",
 
         /// Warm up caches on project load.
-        primeCaches_enable: bool = "true",
+        primeCaches_enabled: bool = "true",
```

Comment:
Should this be `cachePriming`?

Reply:
As in `cachePriming_enabled` or just `cachePriming`? Latter doesn't work due to `primeCaches_numThreads`

Reply:
`cachePriming_enabled`, sorry.

Reply:
Sounds a bit better I guess ye